### PR TITLE
Update ru_RU.lang 30.07.16 v2 version for 1.10

### PR DESCRIPTION
**Pull request by RarogCmex**

### CODE REVIEW

**Comment by modmuss50** [comment]
Thanks :)

File: src/main/resources/assets/techreborn/lang/ru_RU.lang [hunk 1]
```diff
@@ -299,71 +298,71 @@ item.techreborn.part.carbonmesh.name=Углеродный меш
...
+item.techreborn.dustsmall.phosphorous.name=Маленькая кучка фосфорной пыли
+item.techreborn.dustsmall.platinum.name=Маленькая кучка платиновой пыли
+item.techreborn.dustsmall.pyrite.name=Маленькая кучка пиритной пыли
+item.techreborn.dustsmall.pyrope.name=Маленькая кучка пиротной пыли
+item.techreborn.dustsmall.redGarnet.name=Маленькая кучка красногранатной пыли
+item.techreborn.dustsmall.redstone.name=Маленькая кучка красного камня
+item.techreborn.dustsmall.ruby.name=Маленькая кучка рубиновой пыли
+item.techreborn.dustsmall.saltpeter.name=Маленькая кучка селитряной пыли
+item.techreborn.dustsmall.sapphire.name=Маленькая кучка сапфирной пыли
+item.techreborn.dustsmall.sawпыли.name=Маленькая кучка опилочной пыли
```
Comment (PrincessOfEvil) [inline src/main/resources/assets/techreborn/lang/ru_RU.lang:348]: ugh

**Comment by modmuss50** [comment]
The 1.9.4 branch is also the 1.10

File: src/main/resources/assets/techreborn/lang/ru_RU.lang (hunk 1 above)
Comment (modmuss50) [inline src/main/resources/assets/techreborn/lang/ru_RU.lang:348]: Yeah that `item.techreborn.dustsmall.sawпыли.name=Маленькая кучка опилочной пыли` is an issue `all.sawпыли.name=Мале` is not right

**Pull request merged**